spec/models/course_user_spec.rb

```diff
@@ -11,4 +12,17 @@
     it { is_expected.to be_student }
     it { is_expected.not_to be_phantom }
   end
+
+  let!(:instance) { create(:instance) }
+  with_tenant(:instance) do
+    describe '.exp_points' do
```

Comment:
<code>#</code> for instance methods  and <code>.</code> for class methods
http://betterspecs.org/#describe